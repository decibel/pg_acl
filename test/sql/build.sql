\set ECHO none

\i test/pgxntool/psql.sql

BEGIN;

\i sql/acl.sql

\echo TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
