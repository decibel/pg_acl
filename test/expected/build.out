\set ECHO none
 _enum_from_array 
------------------
 
(1 row)

 _enum_from_array 
------------------
 
(1 row)

TRANSACTION INTENTIONALLY LEFT OPEN!
